content_type.go

```diff
@@ -330,7 +330,7 @@ func (service *ContentTypesService) Upsert(spaceID string, ct *ContentType) erro
 	var path string
 	var method string
 
-	if ct.Sys != nil && ct.Sys.CreatedAt != "" {
+	if ct.Sys != nil && ct.Sys.ID != "" {
```

Comment:
I think we should keep `ct.Sts.CreateAt` check along with the `ct.Sts.ID` since `CreateAt` is the only possible way to decide on `update` or `create`.

Reply:
Does it matter?
 * Case 1: Creation of new content type by ID - PUT
 * Case 2: Update of existing content type, with ID specified - PUT

I can't see any other cases specified in the docs, and CreatedAt is not required for this operation on the API. It's not clear to me how this operation would be different when CreatedAt is specified (outside of providing that to the API).

Reply:
I think it's just different opinions here. I'm ok to merge this since the logic is the same at the end. Thanks. ;)